warehouse/admin/views/quarantine.py

```diff
@@ -0,0 +1,42 @@
+# SPDX-License-Identifier: Apache-2.0
+
+"""Admin Views related to Quarantine"""
+from __future__ import annotations
+
+import typing
+
+from pyramid.view import view_config
+from sqlalchemy import select
+
+from warehouse.authnz import Permissions
+from warehouse.packaging.models import LifecycleStatus, Project
+
+if typing.TYPE_CHECKING:
+    from pyramid.request import Request
+
+
+@view_config(
+    route_name="admin.quarantine.list",
+    renderer="warehouse.admin:templates/admin/quarantine/list.html",
+    permission=Permissions.AdminProjectsRead,
+    request_method="GET",
+    uses_session=True,
+    require_csrf=True,
```

Comment:
Does it though isn't this a get only route?

Reply:
bah, hoisted by my own copy&pasta!